errata_parser.rb

```diff
@@ -103,8 +103,8 @@ def write_errata_file(filename, errata, name: '', verbose: false, remove_empty_p
   write_json_file(
     filename,
     data,
-    name: name,
-    verbose: verbose
+    name:,
```

Comment:
don't understand this

Reply:
Shorthand Syntax similar to Javascript:
https://dev.to/baweaver/ruby-3-1-shorthand-hash-syntax-first-impressions-19op

Reply:
maybe pick EnforcedShorthandSyntax: never ?